modelview/templates/modelview/overview.html

```diff
@@ -4,71 +4,57 @@
-    tools as well as senseless printing of assumptions which the users need in
-    exportable formats. The model on which the study is based can be described
-    in a model Fact Sheet. This will link to a framework Fact Sheet (if based on
-    one) and it will be linked from the scenario Fact Sheets that describe the
-    various scenarios of the study. Usually different scenarios with modified
-    input data and assumptions are calculated and compared. As input data is
-    one of the most important information of the scenario, the scenario
-    factsheet will include links to the datasets in a publicly available
-    database (the concept of the OpenEnergy Database is to provide a place to
-    store and publish this data).<br/>
-<br/>
-
-
-
-    The current Factsheets are a result of bringing together existing
-    characterizations and improving it in two workshops and in an online
-    discussion. This discussion is reopened with the growing interest to use the
-    factsheets. The release of Dec. 2017 will include functions to filter the
-    Fact Sheets and to compare them regarding specified characteristics. In 2018
-    we will broaden the view of energy system models and provide the possibility
-    to specify health, social and other aspects.
-<br/>
-<br/>
-
-<sup id="fn1">[1] The standardization process is still going on.</sup>
-<br/>
-<br/>
-</div>
-
-{% endblock %}
\ No newline at end of file
+<h2>Factsheets Overview</h2>
+<p>The Open Energy Platform hosts a series of factsheets – currently we offer
+    <a href="https://openenergy-platform.org/factsheets/models/">Model
+        Factsheets</a>,
+    <a href="https://openenergy-platform.org/factsheets/frameworks/">Framework
+        Factsheets</a> and
+    <a href="https://openenergy-platform.org/factsheets/scenarios/">Scenario
+        Factsheets</a> that help to
+    learn more about the context of the data you can find in the database.
+
+    The Factsheets are a standardized collection and presentation of
+    information about modelling software used in climate
+    and energy system modelling. The use of interactive fields and pre-defined
+    responses is designed to make it easy to insert data and filter
+    for existing entries.
+    For example, you may want to know more about the background of some of the
+    data you found than the metadata attached directly to it can provide. Then
+    factsheets are a helpful place to visit.
+</p>
+<p>
+    Specifically, our current Scenario Factsheets provide some dedicated
+    information about specific scenarios for which projection data is available
+    on the OEP. Please note, however, that their format was
+    developed some time ago and they will undergo substantial updates.
+    In the future, these scenario factsheets will also be succeeded by <a
+        href="https://openenergy-platform.org/factsheets/rdf/study/">Study
+    Factsheets (alpha)</a> that will hold more functionality.
```

Comment:
Suggested change:

These factsheets will be based on RDF and the <a
        href="https://github.com/OpenEnergyPlatform/oekg">Open Energy Knowledge
    Graph (OEKG)</a> which is currently under heavy development. Please note that this is just a preview
    and not a fully functional feature yet.